src/app/locator/qgsactivelayerfeatureslocatorfilter.h

```diff
@@ -63,6 +68,7 @@ class APP_EXPORT QgsActiveLayerFeaturesLocatorFilter : public QgsLocatorFilter
     QgsFeatureIterator mDisplayTitleIterator;
     QgsFeatureIterator mFieldIterator;
     QString mLayerId;
+    bool mLayerIsSpatial = false;
```

Comment:
Why is this a member variable? Looks like it's only used in the one function

Reply:
this is defined in `prepare` and used in `fetchResults`

Reply:
Ah, ok then!